app/views/users/show.html.erb

```diff
@@ -1,4 +1,57 @@
-<div class="container">
-  <h2><%= @user.email %>'s Profile</h2>
+<div class="">
 
+  <div class="row">
+  <div class="col-sm-4">
```

Comment:
Either this should be indented again to match the `col-sm-8` below, or that one should be pulled back, so it's clearer how the DOM hierarchy works. You could also use `_snippet`-style include files so it's easier to parse out the template.